### ISSUE TITLE
Setup CI and notify commiter on failure

### ISSUE THREAD

**Issue by Husterknupp** [Husterknupp/jubilant-todo-tribble#15]
Travis should be good for the job
- https://docs.travis-ci.com/user/languages/java/#Projects-Using-Maven

 - [x] build on every commit
 - [x] build on develop branch
 - [x] build on master branch
 - [x] notify committer on error
 - [x] do not forget fancy badge in README ;-)
 - [-] test if commit on change from failure to success works - Currently there is no mail on build change from failure to success. Because this is not that important. Thats why this ticket will be closed.

**Comment by Bantolomeus** [comment]
@Husterknupp what do you mean with fancy badge in README? I can not find it.

**Comment by Bantolomeus** [comment]
I can not set up Travis, because it is not my project, I only see my 3 projects

**Comment by Bantolomeus** [comment]
@Husterknupp I can not set up Travis, because I have no admin access on your project, only on my own projects

**Comment by Husterknupp** [comment]
find build results here: https://travis-ci.org/Husterknupp/jubilant-todo-tribble/

**Comment by Husterknupp** [comment]
link for the build status badge:
```
[![Build Status](https://travis-ci.org/Husterknupp/jubilant-todo-tribble.svg?branch=master)](https://travis-ci.org/Husterknupp/jubilant-todo-tribble)
```

**Comment by Bantolomeus** [comment]
@Husterknupp do you want to build on every commit on every branch?

**Comment by Husterknupp** [comment]
I want it on develop and on master. On the other branches I'm not sure 🤔 since code on feature branches is mainly maintained by 1 single dev alone. 

What do you say?

**Comment by Bantolomeus** [comment]
Configured a linux and a osx build for travis. The linux build is allowed to fail until we find a solution for both operating systems.

**Comment by Bantolomeus** [comment]
master and develop sounds great to me

**Comment by Bantolomeus** [comment]
new issue to fix failing linux build #21

**Comment by Bantolomeus** [comment]
Currently there is no mail on build change from failure to success. Because this is not that important. Thats why this ticket will be closed.
